examples/hardhat-boilerplate/frontend/src/components/Dapp.js

```diff
@@ -24,8 +24,8 @@ import { NoTokensMessage } from "./NoTokensMessage";
 // Here's a list of network ids https://docs.metamask.io/guide/ethereum-provider.html#properties
 // to use when deploying to other networks.
 //const HARDHAT_NETWORK_ID = '23294'; // Sapphire Mainnet
-//const HARDHAT_NETWORK_ID = '23295'; // Sapphire Testnet
-const HARDHAT_NETWORK_ID = '23293'; // Sapphire Localnet
+const HARDHAT_NETWORK_ID = '23295'; // Sapphire Testnet
```

Comment:
Makes sense. If you are spinning up localnet, you are probably past that beyond.